### PR TITLE
Add media events on wysiwyig

### PR DESCRIPTION
**Pull request by smeeckaert**
I've added two events allowing to change the url of the media parsed and to detect parsed image before sending them to the closure.

### CODE REVIEW

File: framework/classes/tools/wysiwyg.php [hunk 1]
```diff
@@ -186,8 +188,11 @@ public static function parse_medias(&$content, $closure)
         if (!empty($media_matches)) {
             $medias = \Nos\Media\Model_Media::find('all', array('where' => array(array('media_id', 'IN', $media_ids))));
             foreach ($media_matches as $media_match) {
+                $closureImage =\Arr::get($medias, $media_match['id'], null);
```
Comment (shaoshiva) [inline framework/classes/tools/wysiwyg.php]: missing space after "="

Comment (smeeckaert) [inline framework/classes/tools/wysiwyg.php]: Done, sorry.

**Comment by shaoshiva** [comment]
:+1:

**Comment by Meroje** [comment]
:+1:

**Comment by tonyyb** [comment]
:+1:

**Pull request merged**